### PR TITLE
Refs #22493 - documented logging macros

### PR DESCRIPTION
**Pull request by lzap**
It was missing in the initial patch. Since this is a trivial change, I
am not creating extra ticket for this. @ares

### CODE REVIEW

File: config/logging.yaml [hunk 1]
```diff
@@ -56,10 +56,19 @@
...
 #     [%M] method name where the logging request was issued
 #     [%X{request}] request-ID set in HTTP headers, or random UUID
-#     [%X{session}] random ID assigned per session, not available early on during
-#                   request logging, but prevents disclosure of the session ID
-#     [%X{string}] variable set via MDC (::Logging.mdc['string'])
-#     [%x{string}] variable set via NDC (::Logging.ndc['string'])
+#     [%X{session}] random ID assigned per session
+#     [%X{user_login}] currently logged user if present
+#     [%X{org_id}] current organization id if available
+#     [%X{loc_id}] current organization id if available
```
Comment (ares) [inline config/logging.yaml]: current location I guess :-)

**Review by ares — DISMISSED** [review]
one minor thing and good to go (we don't need to wait for tests :-)

**Comment by mmoll** [comment]
Please add `[skip ci]` as second line of the commit message, then (at least some?) tests shouldn't get triggered.

**Comment by lzap** [comment]
Added the keyword, rebased. Nice @mmoll thanks!

File: config/logging.yaml [hunk 2]
```diff
@@ -56,10 +56,19 @@
...
+#     [%X{org_id}] current organization id if available
+#     [%X{loc_id}] current location id if available
+#     [%X{remote_ip}] remote IP
+#     [%x{exception_class}] exception info (only for exception entries)
+#     [%x{exception_message}] exception info (only for exception entries)
+#     [%x{exception_backtrace}] exception info (only for exception entries)
+#     [%X{string}] generic field set via MDC (::Logging.mdc['string'])
+#     [%x{string}] generic field set via NDC (::Logging.ndc['string'])
+#
+#  Some MDC/NDC macros might be available in some contexts and they will render as
```
Comment (mmoll) [inline config/logging.yaml]: should this read "...might NOT be available..."?

Comment (lzap) [reply config/logging.yaml]: What I want to say is that we don't filter out any fields so you can end up with:

```
{ ....
 mdc: {
   "org_name", ""
 }
}
```

But I think it's not relevant, I will remove that.

**Comment by lzap** [comment]
Rebased, the comment was actually irreverent for the last version. It was leftover.

**Review by mmoll — APPROVED** [review]

**Comment by mmoll** [comment]
merged, díky @lzap!

**Pull request merged**